crates/pixi_utils/src/prefix_guard.rs

```diff
@@ -1,11 +1,9 @@
-use std::{
```

Comment:
Would you be able to do this change as a seperate PR?

Reply:
yes and no - 
if I would move it into separate PR , we cannot anymore test reporting multiple clonning of git repositories, for example by using this deps:
```
boost-check = { git = "https://github.com/wolfv/pixi-build-examples", branch = "main", subdirectory = "boost-check" }
bzip2 = { git = "https://github.com/wolfv/pixi-build-examples", branch = "main", subdirectory = "build-recipe/recipes/bzip2" }
```
because we will hit the deadlock.

so I thought that it would be appropriate to do this fix in this PR.

wdyt?